src/dotnet-core-uninstall/Windows/RegistryQuery.cs

```diff
@@ -53,6 +35,30 @@ public virtual IEnumerable<Bundle> GetAllInstalledBundles()
             return wrappedBundles;
         }
 
+        private IEnumerable<RegistryKey> GetNetCoreBundleKeys(RegistryKey uninstallKey)
+        {
+            try
+            {
+                var uninstalls = uninstallKey
+                    .OpenSubKey("SOFTWARE")
+                    .OpenSubKey("Microsoft")
+                    .OpenSubKey("Windows")
+                    .OpenSubKey("CurrentVersion")
+                    .OpenSubKey("Uninstall");
```

Comment:
Here you can make a single call by providing the path
```suggestion
                    .OpenSubKey(@"SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall");
```